packages/template-mrt-reference-app/app/isolation-tests.js

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright (c) 2023, Salesforce, Inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+/* eslint-disable @typescript-eslint/no-var-requires */
+
+const {LambdaClient, InvokeCommand} = require('@aws-sdk/client-lambda')
+const {S3Client, GetObjectCommand} = require('@aws-sdk/client-s3')
+const {CloudWatchLogsClient, PutLogEventsCommand} = require('@aws-sdk/client-cloudwatch-logs')
+
+export const isolationOriginLambdaTest = async (input) => {
+    const client = new LambdaClient()
+    try {
+        await client.send(new InvokeCommand(input))
+    } catch (e) {
+        if (e.name === 'AccessDeniedException') {
+            return true
+        }
+        console.error(e)
+    }
+    console.error('Lambda isolation test failed!')
+    return false
+}
+
+export const isolationS3Test = async (input) => {
+    const client = new S3Client({region: 'us-east-1'})
+    try {
+        await client.send(new GetObjectCommand(input))
+    } catch (e) {
+        if (e.name === 'AccessDenied') {
```

Comment:
confirming this exception is 'AccessDenied' not 'AccessDeniedException' like the other tests?

Reply:
Yes they are different exceptions